services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -113,20 +106,36 @@ public class TargetApplicationSubmissionServiceTest extends AbstractSdtUnitTestB
     /**
      * The mocked ICacheable reference to the error message cache.
      */
+    @Mock
     private ICacheable mockErrorMsgCacheable;
 
+    private static final String TEST_1 = "TEST_1";
+
+    private static final String TARGET_APP_TIMEOUT = "TARGET_APP_TIMEOUT";
+
+    private static final String TARGET_APP_RESP_TIMEOUT = "TARGET_APP_RESP_TIMEOUT";
+
+    private static final String RECEIVED = "Received";
+
+    private static final String FORWARDED = "Forwarded";
+
+    private static final String MCOL_INDV_REQ_DELAY = "MCOL_INDV_REQ_DELAY";
+
+    private static final String TWELVE_THOUSAND = "12000";
+
     /**
      * Method to do any pre-test set-up.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
 
         // Instantiate all the mocked objects and set them in the target application submission service
-        mockIndividualRequestDao = EasyMock.createMock(IIndividualRequestDao.class);
-        mockConsumerGateway = EasyMock.createMock(IConsumerGateway.class);
-        mockCacheable = EasyMock.createMock(ICacheable.class);
-        mockMessageWriter = EasyMock.createMock(IMessageWriter.class);
-        mockErrorMsgCacheable = EasyMock.createMock(ICacheable.class);
+        mockIndividualRequestDao = mock(IIndividualRequestDao.class);
+        mockConsumerGateway = mock(IConsumerGateway.class);
+        mockCacheable = mock(ICacheable.class);
+        mockMessageWriter = mock(IMessageWriter.class);
+        mockErrorMsgCacheable = mock(ICacheable.class);
```

Comment:
This can be removed as the mocks are set up by the mock annotations on the corresponding fields.

Add the Mock annotation to the mockMessageWriter field.

Reply:
Done